### PR TITLE
Generalize model usage in burn-import README

### PR DESCRIPTION
**Pull request by Luni-4**
### Checklist

- [x] Confirm that `run-checks` script has been executed.

### Changes

This PR generalizes model usage in `burn-import` README

### Testing

_Describe how these changes have been tested._

### CODE REVIEW

**Review by nathanielsimard — APPROVED** [review]

File: burn-import/README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Follow these steps to import an ONNX model into your Burn project:
 
    ```rust
    pub mod mnist {
-       include!(concat!(env!("OUT_DIR"), "/model/mnist.rs"));
+       include!(concat!(env!("OUT_DIR"), "/model/model_name.rs"));
    }
    ```
```
Comment (nathanielsimard) [inline burn-import/README.md:44]: Wrong module name!

**Review by nathanielsimard — CHANGES_REQUESTED** [review]

File: burn-import/README.md (hunk 1 above)
Comment (antimora) [reply burn-import/README.md:44]: It should be the same as onnx file name.

Comment (Luni-4) [reply burn-import/README.md:44]: Ok, so can we leave as it is now?

Comment (nathanielsimard) [reply burn-import/README.md:44]: I would put model_name

**Review by nathanielsimard — APPROVED** [review]

**Pull request merged**